src/object/cli_obj.c

```diff
@@ -908,8 +908,11 @@ struct obj_auxi_args {
 	int				 result;
 	d_list_t			 shard_task_head;
 	tse_task_t			*obj_task;
-	struct daos_obj_shard_tgt	*fw_shard_tgts;
-	uint32_t			 fw_cnt;
+	struct daos_obj_shard_tgt	*fw_shard_tgts_inline;
+	struct daos_obj_shard_tgt	**fw_shard_tgts;
+	uint32_t			*fw_cnt;
+	uint32_t			 fw_cnt_inline;
+	uint32_t			 fw_rdg_count;
```

Comment:
some comments about what it is is helpful... thanks

Reply:
I will update the patch for that.